### ISSUE TITLE
unable to skip dependency with --assume-installed / --nodeps

### ISSUE THREAD

**Issue by irgendwr** [Jguer/yay#1137]
#### Affected Version

yay v9.4.2 - libalpm v12.0.1

#### Issue

I am unable to skip a given dependency using yay with the `--assume-installed` flag.
In #722 it was mentioned that yay doesn't understand the flag. It looks like that's still the case.

> yay does not actually understand this flag.

Similarly the `--nodeps` flag is also not understood by yay.

If there is another way to skip dependencies then please let me know, thanks.

#### Steps to reproduce

1. `yay --assume-installed pyqt5-common multibootusb`
2. Choose `1`

Note: This example no longer produces the same output because the `pyqt5-common` dependency was dropped. I don't have another example at the moment but the issue still exists of course.

#### Output

```
~ >>> yay --assume-installed pyqt5-common multibootusb
2 aur/multibootusb-git 20181002.dec46ef-1 (+13 2.05%) (Out-of-date 2019-12-23) 
    Boot multiple live Linux distros from a usb flash drive.
1 aur/multibootusb 9.2.0-3 (+104 1.04%) (Out-of-date 2019-12-16) 
    Boot multiple live Linux distros from a usb flash drive.
==> Packages to install (eg: 1 2 3, 1-3 or ^4)
==> 1
==> Error: Could not find all required packages:
    pyqt5-common (Wanted by: multibootusb)
```

**Comment by ftsell** [comment]
I currently have the same problem in the following constellation:
(I know it requires a whole bunch of dependencies, sorry about that)

`yay -S --assume-installed ros-melodic-rqt-image-view ros-melodic-desktop-full`

the package `ros-melodic-rqt-image-view` is currently not compiling so I wish to skip it until a change is released.

**Comment by Jguer** [comment]
I can confirm this issue

**Comment by YourPsychiatrist** [comment]
Any updates here? :) Would really need this

**Comment by jclsn** [comment]
I read somehwere that yay is abandoned and you should use paru instead, which is a maintained fork

**Comment by irgendwr** [comment]
> I read somehwere that yay is abandoned and you should use paru instead, which is a maintained fork

\**checks commits\**  Last commit three days ago... yep looks totally abandoned :p
Also, paru is not a fork of yay. But this is all off-topic anyway.
*(Edit: I think you are referring to [this](https://www.reddit.com/r/archlinux/comments/jjn1c1/paru_v100_and_stepping_away_from_yay/))*

Back on topic: maybe I'll give this a try if I find the time for it.

**Comment by jclsn** [comment]
Paru did the trick for me though. Have not used it until now and the name is also not as catchy as yay, but well it worked.

**Comment by irgendwr** [comment]
Fixed in 7dda0fa2f84cd1e32568ec583c3dca4502f0f8df + 186c7a54fea59be667bcf5c9d98ced9e1955f433. Thanks for merging and correcting my patch @Jguer :)

For those that use this flag:
When installing from AUR you still need to manually edit PKGBUILD files to remove/skip dependencies as far as I know.
